tests/test_conversion.py

```diff
@@ -146,9 +146,10 @@ def _test_to_dataframe(self, mdoc, test):
             test.ct_data_frame_rows,
             f"The pandas data frame has less elements than the orginal one for {test.filename}",
         )
-        df.to_csv(test.get_out_file("roundtrip.tsv"), sep="\t")
-        # data = pd.read_csv(test.get_out_file("roundtrip.tsv"), sep="\t")
-        data = parse_sssom_table(test.get_out_file("roundtrip.tsv")).df
+        path = test.get_out_file("roundtrip.tsv")
```

Comment:
This makes it so the prefix map is output with the pandas dataframe. Before, none of the test data had properly loaded prefix maps, so it was checking two empty prefix maps were the same. This update makes tests properly output the prefix map with the data.

Reply:
Thanks!